tests/functional/cylc-ping/04-check-keys-remote.t

```diff
@@ -53,26 +53,13 @@ suite_run_ok "${TEST_NAME_BASE}-run" cylc play \
     --debug
 
 KEYS_FILE="$(cylc cat-log -m p "$SUITE_NAME" 'keys.1' -f job-find-out)"
-if [[ "$CYLC_TEST_PLATFORM" == *shared* ]]; then
-    cmp_ok "$KEYS_FILE" <<__OUT__
+cmp_ok "$KEYS_FILE" <<__OUT__
 client.key_secret
 client_${CYLC_TEST_INSTALL_TARGET}.key
 server.key
-server.key_secret
 __OUT__
-else
-    cmp_ok "$KEYS_FILE" <<__OUT__
-client.key_secret
-client_${CYLC_TEST_INSTALL_TARGET}.key
-server.key
-__OUT__
-fi
 
-if [[ "$CYLC_TEST_PLATFORM" == *shared* ]]; then
-    skip 1
-else
-    grep_ok "Removing authentication keys and contact file from remote: \"${CYLC_TEST_INSTALL_TARGET}\"" "${SUITE_RUN_DIR}/log/suite/log"
-fi
+grep_ok "Removing authentication keys and contact file from remote: \"${CYLC_TEST_INSTALL_TARGET}\"" "${SUITE_RUN_DIR}/log/suite/log"
```

Comment:
Why has the shared filesystem test been removed?

Reply:
Because there's already a test for it, see the other file https://github.com/cylc/cylc-flow/pull/4089/files#diff-18ed6f53d34c84b28d36af7117a10837b94185f6007c54a973675affd1a44052

Reply:
Ok, can we delete